CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## Version 0.8 (2023-11-01)
```

Comment:
Bump to 0.7.1 sufficient according to https://pvp.haskell.org/ paragraphs 2 and 8.